src/lemlib/movements/purepursuit.cpp

```diff
@@ -128,6 +129,6 @@ std::pair<int, int> PurePursuit::update(Pose pose) {
         rightVel *= -1;
     }
 
-    return {std::round(leftVel), std::round(rightVel)};
+    return {std::round(leftVel.convert(rpm)), std::round(rightVel.convert(rpm))}; // todo test
```

Comment:
```suggestion
    return {std::round(leftVel.convert(inps)), std::round(rightVel.convert(inps))}; // todo test
```
Convert to correct units